configure

```diff
@@ -10,6 +10,7 @@ command -v python3.8 >/dev/null && exec python3.8 "$0" "$@"
 command -v python3.7 >/dev/null && exec python3.7 "$0" "$@"
 command -v python3.6 >/dev/null && exec python3.6 "$0" "$@"
 command -v python3.5 >/dev/null && exec python3.5 "$0" "$@"
+command -v python3 >/dev/null && exec python3 "$0" "$@"
```

Comment:
@cclauss Would this be acceptable?

```suggestion
command -v python3 >/dev/null && python3 -c 'import sys; assert sys.version_info >= (3,5)' > /dev/null && exec python3 "$0" "$@"
```

Reply:
Although I guess I kind of wonder if this actually covers anything we don't already cover.

@oleavr It would be very useful to know the output of `python3 --version` on your system (assuming this PR is a bug fix for your own setup).

Reply:
We could also just print a warning from `configure.py` itself, right? We’re already doing that for a the compiler/assembler/etc. versions as well

Reply:
@Trott The output is:

```
% python3 --version
Python 3.8.2
```

This is the Xcode-provided Python 3 on a macOS system. Xcode doesn't create a `python-3.8` symlink in `/usr/bin`, there's only `/usr/bin/python3`.